Javadoc errors in generated code cause release to fail

```bash
$ mvn release:prepare
...
    [ERROR] /working/bdg-formats-upstream/target/generated-sources/avro/org/bdgenomics/formats/avro/TranscriptEffect.java:40: warning: no @return
    [ERROR]   public static BinaryMessageDecoder<TranscriptEffect> createDecoder(SchemaStore resolver) {
    [ERROR]                                                        ^
    [ERROR]
    [ERROR] Command line was: /Library/Java/JavaVirtualMachines/jdk1.8.0_191.jdk/Contents/Home/jre/../bin/javadoc @options @packages
...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
```

--- a/pom.xml
+++ b/pom.xml
@@ -79,7 +79,7 @@
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>3.0.1</version>
           <configuration>
-            <additionalparam>-Xdoclint:none</additionalparam>
+	    <additionalJOptions>-Xdoclint:none</additionalJOptions>
           </configuration>
         </plugin>
       </plugins>
@@ -148,7 +148,7 @@
             <artifactId>maven-javadoc-plugin</artifactId>
             <version>3.0.1</version>
             <configuration>
-              <additionalparam>-Xdoclint:none</additionalparam>
+	      <additionalJOptions>-Xdoclint:none</additionalJOptions>
             </configuration>
             <executions>
               <execution>

